dust_extinction/helpers.py

```diff
@@ -4,6 +4,8 @@
 from scipy.special import comb
 import astropy.units as u
 
+from .warnings import SpectralUnitsWarning
```

Comment:
I have been using absolute paths instead of relative throughout this package.  So, please change this to use absolute imports instead of relative for consistency.

Reply:
I retract this comment.  Clearly both have been used in this repository.